scripts/quests/sandoria/Healing_The_Land.lua

```diff
@@ -0,0 +1,96 @@
+-----------------------------------
+-- Healing the Land
+-----------------------------------
+-- !addquest 0 82
+-- Eperdur: !pos 129 -6 96 231
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+local ID = require("scripts/zones/Northern_San_dOria/IDs")
+local ID2 = require("scripts/zones/Gusgen_Mines/IDs")
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.HEALING_THE_LAND)
+
+quest.reward =
+{
+    fame = 30,
+    fameArea = xi.quest.fame_area.SANDORIA,
+    item = xi.items.SCROLL_OF_TELEPORT_HOLLA,
+    title = xi.title.PILGRIM_TO_HOLLA,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.SANDORIA) >= 4 and
+                player:getMainLvl() >= 10
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Eperdur'] = quest:progressEvent(681),
+
+            onEventFinish =
+            {
+                [681] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    npcUtil.giveKeyItem(player, xi.ki.SEAL_OF_BANISHING)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Eperdur'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:hasKeyItem(xi.ki.SEAL_OF_BANISHING) then
+                        return quest:event(682)
+                    elseif not player:hasKeyItem(xi.ki.SEAL_OF_BANISHING) then
+                        return quest:progressEvent(683)
+                    else
+                        return quest:event(684)
```

Comment:
I don't think this event will ever be reached. Is this just his regular default action?

Reply:
684 should go as a replaceDefault() in QUEST_COMPLETE in healing_the_land.lua

Reply:
Makes sense, i keep forgetting about that function. What I ought to do is build a default guide quest that breaks down each section and figure out what goes where.